.swc

```diff
@@ -1,13 +1,13 @@
 {
   "jsc": {
-    "target": "es2018",
+    "target": "es2021",
```

Comment:
This aligns with Node 16, which is the minimum requirement across our other apps